### DISCUSSION TITLE
Question on Sample Web-Viewer

### DISCUSSION THREAD

**Discussion by dvmalkan** [iTwin/itwinjs-core#1589] [General] [answered]
Got & ran the 'Getting Started' Sample: https://developer.bentley.com/tutorials/web-application-quick-start/

The primary relevant control loaded (in file App.tsx) is:
<Viewer
            contextId={process.env.IMJS_CONTEXT_ID ?? ""}
            iModelId={process.env.IMJS_IMODEL_ID ?? ""}
            authConfig={{ oidcClient: AuthorizationClient.oidcClient }}
 />

Just this control gives a lot more than I expected: The core Viewer + its viewing control tools + Components Tree + Sectioning & other Tools.

How can I create a View with only the basic Viewing Controls? Are there any settings to the Viewer control?

Deepak.

**Comment by aruniverse** [comment, marked as answer]
Hi Deepak,

There is a prop on the viewer, `defaultUiConfig` ([see here](https://github.com/imodeljs/itwin-viewer/tree/master/packages/modules/itwin-viewer-react#optional)), that you can set to disable a lot of the functionality currently baked into the base viewer.

**Reply by dvmalkan** [reply]
Thx.

I can disable all but the 'corner item'. Seems to me that there should be a way to simply disable the corner item, rather than replace it with another item.

The goal is to have the ability to insert an iModel Viewer as a simple control that can be seamlessly integrated into an existing Web Application. Currently it seems to be configured such that the iModel Viewer is the primary Web Application (that can be extended), rather than a simple React Control.

Deepak.

**Reply by kckst8** [reply]
Hi Deepak. You can use : "{cornerItem: {hideDefault: true}}" to hide the corner item with no replacement. 

The iTwin Viewer is just a React component. It is not an application. Please let us know where you are having trouble integrating it into your application and perhaps we can help. The README that @aruniverse posted is your best bet for customization options.

**Reply by DeepakMalkan** [reply]
Thx. That works.

That the cornerItem can be hidden is not clear from the documentation, though Intellisense in Code does show the 'hideDefault' option.

** cornerItem - replace the default backstage navigation button with a new item **

Deepak.
